rubocop_all.yml

```diff
@@ -6,4 +6,5 @@ AllCops:
     - '**/vendor/**/*'
```

Comment:
hey bro can you add this one

`  - 'vendor/**/*'`

Reply:
Do you have ruby/haml codes in it?

Reply:
never mind bro ignore me.